.mvn/jvm.config

```diff
@@ -1,3 +0,0 @@
--Dhttp.keepAlive=false
```

Comment:
Should this maven wrapper file be removed as part of this pr?

Reply:
Done